src/helpers/guest.js

```diff
@@ -0,0 +1,19 @@
+import {guest} from "../server/models";
+import asyncHanlder from "express-async-handler";
+
+exports.createGuest = asyncHanlder(async(req, res) => {
+   
+const {name, email, phone, gender} = req.body;
+try {
+const newGuest = await guest.create({
+    name, 
+    email,
+    phone,
+    gender
+});
+res.status(201).json({success: true, statusCode: 201, message: "Guest Information successfully saved", Guest: newGuest});
```

Comment:
write a function to handle all your response.

Reply:
I have written a function that handles all the responses.